cog/admin_role.py

```diff
@@ -55,14 +55,22 @@ def __reward(uid, user_name, bouns_type, bouns):
             write(uid, bouns_type, current_point + bouns, cursor)
             end(connection, cursor)
             print(f"{uid} {user_name} get {bouns} {bouns_type} by Gift")
-
+        # 存資料庫存取按鈕屬性(包括獎勵類型、數量)
+        def __readDB(self, btnid):
```

Comment:
(From Pylint)

Didn't meet the naming convension.

Could rename to `__read_db` instead.